src/bin/build.rs

```diff
@@ -43,6 +44,7 @@ Options:
     -h, --help                   Print this message
     -p SPEC, --package SPEC ...  Package to build
     --all                        Build all packages in the workspace
+    --exclude SPEC ...           Exclude packages from the build
```

Comment:
>Is there someone working on it already? 

I've tried to do that, but haven't arrived at something very satisfying :) 

It would be super if you would be able to pull this off!